cypress/e2e/news/createReadDeleteNewsInAdvance.feature

```diff
@@ -0,0 +1,49 @@
+@stable_test
+@regression_test
+Feature:  News - To read a news on the respective dashboards
+
+    As a teacher I want to read the news shown on the dashboard so that I'm informed about the latest news
+
+    Scenario: User creates news with time options, reads them and deletes them
+        # as a pre-condition create users for author and reader
+        Given I am logged in as a '<news_reader>' at '<namespace>'
```

Comment:
we should use '<teacher>' or '<student>' to be specific here, like in other features.

Reply:
This test would be possible for different roles, so I want to keep it flexibel and not to define a special role.